custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -141,7 +141,7 @@ def predict(self, **kwargs):
         end_predict = time.time()
         execution_time_ms = (end_predict - start_predict) * 1000
         self.monitor(kwargs, predictions, execution_time_ms)
-        return predictions
+        return predictions, extra_df
 
     @abstractmethod
     def _predict(self, **kwargs):
```

Comment:
please add return requirements here with a dataclass separate from the other one

```
@dataclass
class DetailedPredictionResult:
    predictions: pd.DataFrame
    stuuuuuuuuuf: pd.DataFrame
    prediction_columns:  pd.Series?  List?
```

Reply:
Done